roboteam_ai/src/control/ballHandling/BallHandlePosControl.cpp

```diff
@@ -236,9 +255,9 @@ RobotCommand BallHandlePosControl::goToBall(const Vector2 &targetBallPos, Travel
     bool ballIsMoving = ball->vel.length2() > minVelForMovingball*minVelForMovingball;
 
     if (ballIsMoving) {
-        return goToMovingBall();
+        return updatePID(xGoToBallPID, yGoToBallPID, goToMovingBall());
     }
-    return goToIdleBall(targetBallPos, travelStrategy, ballIsFarFromTarget);
+    return updatePID(xGoToBallPID, yGoToBallPID, goToIdleBall(targetBallPos, travelStrategy, ballIsFarFromTarget));
```

Comment:
how would the travelstrategy ever matter for updating the PID?

Reply:
I should put this on two lines.  'goToIdleBall' is a different function call